src/main.py

```diff
@@ -29,8 +30,7 @@ def outline(year, lang):
 @app.route('/<lang>/<year>/contributors')
 @validate
 def contributors(year, lang):
-    # TODO: Get contributor data and pass into the template.
-    return render_template('%s/%s/contributors.html' % (lang, year), contributors={})
+    return render_template('%s/%s/contributors.html' % (lang, year), contributors=contributor_util.get_contributors())
```

Comment:
Ah you have `contributor_util` but it's imported as `contributors_util`. Sorry if I suggested the wrong spelling!

Reply:
No more error, but now it's just a blank page. I inspected it and the div with the id="main" is empty. Do we need to import the data? I don't see anything that refers to any of the other files.

Reply:
In contributors.py, should "contributors_file" be "contributors_util"?

Reply:
I just tested your PR locally and found the issue. I'll walk you through my debug process so you can see what I did and how I identified the issues.

First, I wanted to inspect what the value of the `contributors` object is as it's passed to the template. So I added this logging statement in `main.py`:

```py
def contributors(year, lang):
    contributors = contributors_util.get_contributors()
    logging.debug('Contributors: %r' % contributors)
    return render_template('%s/%s/contributors.html' % (lang, year), contributors=contributors)
```

Then after reloading the page I see this line in the terminal:

```
DEBUG:root:Contributors: {}
```

So the object is not populated with the contents of the JSON file. Looking in `contributors.py` I noticed that the invocation on line 17 is calling `get_contributors` instead of `update_contributors`.

After I fixed that I reloaded the page again and the page was still blank. The JSON was being logged as expected this time:

```
DEBUG:root:Contributors: {'teams': {'developers': {'name': 'Web Developers'}, 'brainstormers': {'name': 'Brainstormers'}, 'authors': {'name': 'Chapter Authors'}, 'reviewers': {'name': 'Peer Reviewers'}, 'analysts': {'name': 'Data Analysts'}, 'designers': {'name': 'Web Designers'}, 'translators': {'name': 'Content Translators'}, 'editors': {'name': 'Editors'}}, 'contributors': {'argyleink': {'name': 'Adam Argyle', 'github': 'https://github.com/argyleink', 'teams': ['brainstormers']}, 'kjlarson': {'name': 'Kari Larson', 'github': 'https://github.com/KJLarson', 'teams': ['developers']}, 'rviscomi': {'name': 'Rick Viscomi', 'github': 'https://github.com/rviscomi', 'teams': ['brainstormers', 'developers']}}}
```

But looking in the source HTML I see that two sections are being rendered with empty fields:

![image](https://user-images.githubusercontent.com/1120896/60289086-a08b6980-98e3-11e9-81b6-39a1fb784a27.png)

The name of the JSON object is `contributors` but it also has properties named `teams` and `contributors`. So it's rendering a section for each of these and not the individual contributors. I modified the logic in the template to this:

```html
{% for id, contributor in contributors.contributors.items() %}
<section>
  <h3>{{ contributor.name }}</h3>
  <ul>
    {% for team in contributor.teams %}
    <li>{{ contributors.teams[team].name }}</li>
    {% endfor %}
  </ul>
</section>
{% endfor %}
```

The major changes were:

- iterating over the `contributors` object within the variable named `contributors`
- calling `.items()` on the JSON object so we can access its key/value separately
- assigning the key/value pair `id, contributor` respectively
- referencing the `teams` object within the `contributors` variable

Now I'm seeing the page render as expected:

![image](https://user-images.githubusercontent.com/1120896/60289499-8e5dfb00-98e4-11e9-96b0-0bbb5cd7abc1.png)

So go ahead and make the fixes to `contributors.py` and `contributors.html`. No need to commit the logging code, that's just for local debugging.

Reply:
I'm trying to recreate your process, and when I try to call update_contributors on line 17, I get this error:
```
line 13, in update_contributors
    with open('config/contributors.json', 'r') as contributors_file:
FileNotFoundError: [Errno 2] No such file or directory: 'config/contributors.json'
```
I cannot figure out what is happening here. I double-checked the spelling and path a thousand times, and they seem right. Did you run into anything like this?

Reply:
I added src/ in front of config/contributors.json and that seemed to work. That confuses me. I thought that when we use a relative path, we can just type a name that is on a sibling level in the directory tree. Doesn't this go up and then back down? Does it work differently here?

Reply:
When we deploy the app to production, it takes everything under `src` and runs it on App Engine. So there will be no `src` directory.

When you're running it locally are you running `python src/main.py` or from the src directory are you running `python main.py`? The latter is the correct way. We should update our README.

Reply:
Aha!